statik/statik.go

```diff
@@ -0,0 +1,10 @@
+package statik
```

Comment:
@codysoyland removed `statik` in #1363, so this file shouldn't be here.
I do however see that `github.com/rakyll/statik` is still in `Gopkg.lock`, so we need to get rid of that as well.